traffic-access-control.md

```diff
@@ -73,6 +73,49 @@ sources:
 - kind: ServiceAccount
   name: prometheus
   namespace: default
+- kind: Service
+  name: foo.mesh
+- kind: Service
+  name: bar.mesh
+- kind: Service
+  name: baz.mesh
+```
+
+```yaml
+# Register a service 'foo.mesh'
```

Comment:
@delqn  - this has the facility to use different source types (e.g. a Kubernetes Service, DNS name, Azure resource name). But does it allow users to also use these types for the destination? This will allow creation of access policies for applications running in VMs, for e.g.